### ISSUE TITLE
editing fails after editor closes

### ISSUE THREAD

**Issue by dankelley** [dankelley/nota#23]
```
Press ENTER or type command to continue
Traceback (most recent call last):
  File "/usr/local/bin/nota", line 10, in <module>
    sys.exit(nota())
  File "/usr/local/lib/python3.7/site-packages/nota/main.py", line 454, in nota
    nota.edit(args.edit)
  File "/usr/local/lib/python3.7/site-packages/nota/notaclass.py", line 717, in edit
    self.keyword_hookup(noteId, ee["keywords"])
  File "/usr/local/lib/python3.7/site-packages/nota/notaclass.py", line 518, in keyword_hookup
    keyword = keyword.decode('utf-8')
AttributeError: 'str' object has no attribute 'decode'
```

**Comment by dankelley** [comment]
Fixed in "master" branch, commit 09ee6e97555ee6c7b69fc7bf57a99896a0882772

**Comment by dankelley** [comment]
Fixed
